puffin/src/lib.rs

```diff
@@ -553,9 +553,9 @@ pub fn now_ns() -> NanoSecond {
 
     // This can maybe be optimized
 
-    #[cfg(not(feature = "web"))]
+    #[cfg(not(target_arch = "wasm32"))]
     use std::time::Instant;
-    #[cfg(feature = "web")]
+    #[cfg(target_arch = "wasm32")]
```

Comment:
@emilk this won't work, we need to use `web_time` only when the web feature is enabled, so we don't use it on `wasm32-unknown-unknown` otherwise (which supports neither `web_time` nor `std::time::Instant`

Reply:
The whole function is wrapped in `#[cfg(any(not(target_arch = "wasm32"), feature = "web"))]`, so I think we should be fine?

Reply:
I added a CI step to check that `--target wasm32-unknown-unknown` also passes _without_ the `web` feature flag.

Reply:
ah yes that should be fine then, great!